src/order/complete_lattice.lean

```diff
@@ -926,6 +926,10 @@ lemma supr_subtype' {p : ι → Prop} {f : ∀ i, p i → α} :
   (⨆ i (h : p i), f i h) = (⨆ x : subtype p, f x x.property) :=
 (@supr_subtype _ _ _ p (λ x, f x.val x.property)).symm
 
+lemma supr_subtype'' {ι} (s : set ι) (f : ι → α) :
```

Comment:
Perhaps
```suggestion
lemma supr_coe_subtype {ι} (s : set ι) (f : ι → α) :
```
since `(⨆ i : s, f i)` is coercing `i`?

Reply:
I would prefer to separate feature PRs from refactor PRs. Probably we need a PR that uniformizes lemma names for these lemmas and similar lemmas about `forall`/`exists`/`Union`/`Inter`.